docs/oscar_references.bib

```diff
@@ -2230,6 +2230,21 @@ @Article{Rin13
   zbmath        = {6143064}
 }
 
+@article {Sau14,
+    AUTHOR = {Saunders, Neil},
+     TITLE = {Minimal faithful permutation degrees for irreducible {C}oxeter
```

Comment:
```suggestion
     TITLE = {Minimal faithful permutation degrees for irreducible Coxeter
```
our bibtex parse does not like parentheses inside of fields. But that's no problem as it is case sensitive (in contrast to most latex implementations)